### ISSUE TITLE
Incompatible latest pyCountry

### ISSUE THREAD

**Issue by croqaz** [miso-belica/sumy#116]
Hi ! The latest version of pycountry~=18.12.8 is incompatible with Sumy.

How to reproduce:
```
pip install pycountry~=18.12.8 -U
python
>>> from sumy.nlp.tokenizers import Tokenizer
>>> Tokenizer('english')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/xxx/.pyenv/versions/env3.6/lib/python3.6/site-packages/sumy/nlp/tokenizers.py", line 65, in __init__
    language = normalize_language(language)
  File "/Users/xxx/.pyenv/versions/env3.6/lib/python3.6/site-packages/sumy/utils.py", line 28, in normalize_language
    return language.name.lower()
AttributeError: 'NoneType' object has no attribute 'name'
```

For versions pycountry~=18.5.26, the same lines:
```
python
>>> from sumy.nlp.tokenizers import Tokenizer
>>> Tokenizer('english')
<sumy.nlp.tokenizers.Tokenizer object at 0x10xxxxx>
```

I'm looking into a solution for this and I'll create a PR.

Unless somebody already has a solution for this and in this case, please do share 🙇

**Comment by croqaz** [comment]
### Update

This can fix the problem in utils.py / normalize_language(language):

>>> languages.get(name='English')
or
>>> languages.get(alpha_2='en')

Now it only works with: normalize_language('en')

Before this, we had:

>>> normalize_language('english')
'english'
>>> normalize_language('en')
'english'

Not sure where it's the best place to fix this: in utils, or in the classes using the normalize_language function?

**Comment by miso-belica** [comment]
Hi @croqaz, thanks for the detailed report and sorry for the very late response. This was solved in https://github.com/miso-belica/sumy/pull/118. I released Sumy v0.8 yesterday so please try to test the new version with all the fixes :)
